client/src/components/HistoryViewer/HistoryViewerToolbar.js

```diff
@@ -57,11 +49,14 @@ class HistoryViewerToolbar extends Component {
       rollbackMessage,
       typeName,
     } = this.props;
-    const { isReverting, revertTooltipOpen } = this.state;
+    const { isReverting } = this.state;
 
-    const revertButtonTitle = isReverting
-      ? i18n._t('HistoryViewerToolbar.REVERT_IN_PROGRESS', 'Revert in progress...')
-      : i18n._t('HistoryViewerToolbar.REVERT_UNAVAILABLE', 'Unavailable for the current version');
+    let revertButtonTitle = '';
+    if (isReverting) {
+      revertButtonTitle = i18n._t('HistoryViewerToolbar.REVERT_IN_PROGRESS', 'Revert in progress...');
+    } else if (isLatestVersion) {
+      revertButtonTitle = i18n._t('HistoryViewerToolbar.REVERT_UNAVAILABLE', 'Unavailable for the current version');
+    }
```

Comment:
Without this, we end up with *two* tooltips when hovering over the revert button for the history details of a related child record.